CHANGELOG.md

```diff
@@ -30,7 +30,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- Bumped clustertest to `v0.1.1` to fix `GITHUB_TOKEN` issue
+- Bumped clustertest to `v0.1.1` to fix `GITHUB_TOKEN` issue.
+- Bumped clustertest to `v0.2.0` to use custom app values.
```

Comment:
This needs moving up to the unreleased section

Reply:
Fixed